contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,131 @@
+
+You can also use subqueries, joins, table expressions, arrays, aggregations, lambdas and a whole host of other neat SQL features, including aggregations.
+
+Whether you're a technical product manager who needs multi-property breakdowns, or an engineer wishing to dissect data in more nuanced ways, HogQL unlocks a whole new world of insights to PostHog users.
+
+We recommend checking the docs for a more detailed [explanation of ClickHouse SQL functions which are supported in HogQL](/manual/hogql).
+
+## HogQL breakdowns
+
+![HogQL breakdowns](../images/blog/array/hog_breakdown.mp4)
+
+<Caption>Using a `properties.$browser_version` HogQL expression to breakdown results</Caption>
+
+A frequent request from teams using PostHog has been the ability to breakdown insight results across multiple properties. To accommodate this, we've added the ability to use HogQL expressions within a breakdown. 
+
+In other words, you can now break down results however you like – including by multiple properties at once. Want to break down new signups by both pricing tier _and_ overall usage? Now you can.
+
+> **Ready to get started?** Read our tutorial on [how to use HogQL for running advanced breakdowns](docs/tutorials/hogql-breakdowns)
+
+## HogQL filters
+
+![HogQL filter](../images/blog/array/hog_filters.mp4)
+
+<Caption>Using a `properties.$screen_width < properties.$screen_height` HogQL expression as a filter</Caption>
+
+Oh, look. You can use HogQL in filters too. Useful for _filtering_ by multiple properties!
+
+## HogQL aggregations
+![HogQL aggregations](../images/blog/array/hog_aggregate.mp4)
+<Caption>Using a `properties.$session_id` HogQL expression to aggregate by unique sessions</Caption>
+
+_And_ you can use HogQL to aggregate results in a funnel too. Is there anything HogQL can't do? Probably, yes, but we'll enjoy finding out for sure. 
+
+## SQL insights
+![PostHog SQL insights](../images/features/hogql/pinepple_sql.gif)
+<Caption>Using an SQL insight to run the query below</Caption>
+
+If you're familiar with SQL, the easiest and most powerful way to leverage it within PostHog is via the new SQL insight type. This gives you direct SQL access to your data in PostHog, so you can create custom tables, visualization and more. 
+
+SQL insights are especially useful for when you need to do exotic or novel analysis that isn't immediately possible with existing insight types. For example, while you can use a retention insight to discover which product features keep users coming back for more, you could use an SQL insight to find which users keep coming back and identify outliers. 
+
+We can't wait to hear your feedback, and see what's possible with SQL access in PostHog. We're confident it will give engineers a unique edge, and better information, than other analytics tools.
```

Comment:
```suggestion
We can't wait to hear [your feedback](http://app.posthog.com/home#supportModal), and see what's possible with SQL access in PostHog. We're confident it will give engineers a unique edge, and better information, than other analytics tools.
```